NuGet.config

```diff
@@ -6,5 +6,6 @@
     <add key="azure_app_service_staging" value="https://www.myget.org/F/azure-appservice-staging/api/v2" />
     <add key="buildTools" value="https://www.myget.org/F/30de4ee06dd54956a82013fa17a3accb/" />
     <add key="AspNetVNext" value="https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json" />
+    <add key="Microsoft.Azure.Functions.PowerShellWorker" value="https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/Microsoft.Azure.Functions.PowerShellWorker/nuget/v3/index.json" />
```

Comment:
Are you placing both packages `Microsoft.Azure.Functions.PowerShellWorker.PS6` and `Microsoft.Azure.Functions.PowerShellWorker.PS7` at this feed?

Reply:
> Are you placing both packages Microsoft.Azure.Functions.PowerShellWorker.PS6 and Microsoft.Azure.Functions.PowerShellWorker.PS7 at this feed?

Looks like all checks passed, so I am assuming that the answer is yes.

Reply:
Actually, the PS7 package is not on the new feed, but it is still published to the old MyGet, this is why the build succeeds. I will put it to the new feed later today, for consistency. But this does not change anything for this PR.

Reply:
Sounds good. Thanks Anatoli.